static/css/biomedical/disease.scss

```diff
@@ -60,3 +61,26 @@ tbody tr:nth-child(even) {
   stroke-width: 1.5;
   border: 3px solid #000;
 }
+
+.map {
```

Comment:
do you still need all these style changes after importing the tiles sheet?

Reply:
removed the unnecessary css elements!